cherrybomb-engine/src/scan/active/http_client/req.rs

```diff
@@ -196,7 +203,10 @@ impl AttackRequest {
             .collect()
     }
     pub async fn send_request_with_response(&self) -> (String, bool) {
-        let client = reqwest::Client::new();
+        let client = Client::builder()
+            .danger_accept_invalid_certs(self.ignore_tls_errors)
```

Comment:
Are you sure that the boolean matches? I mean will it get the desired outcome or is it the opposite?